samples/dnn/human_parsing.cpp

```diff
@@ -0,0 +1,100 @@
+//
+// this sample demonstrates parsing(segmenting) human body parts from an image using opencv's dnn,
+// based on https://github.com/Engineering-Course/LIP_JPPNet
+//
+// get the pretrained model from: https://www.dropbox.com/s/qag9vzambhhkvxr/lip_jppnet_384.pb?dl=0
+//
+
+#include <opencv2/core.hpp>
+#include <opencv2/dnn.hpp>
+#include <opencv2/highgui.hpp>
+#include <opencv2/imgproc.hpp>
+#include <opencv2/imgcodecs.hpp>
+using namespace cv;
+
+static Mat parse_human(const std::string &image, const std::string &model, int backend=dnn::DNN_BACKEND_DEFAULT, int target=dnn::DNN_TARGET_CPU) {
+    // this network expects an image and a flipped copy
+    Mat input = imread(image);
```

Comment:
I think it’s better to read image in `main()` because we want to use `parse_human()` in other samples